assets/css/src/components-form-editor/_block-editor.scss

```diff
@@ -22,6 +22,11 @@
   max-width: 100%;
 }
 
+// Fix for Block toolbar hides under the main top bar
+.interface-interface-skeleton__body {
+  margin-top: 2rem;
```

Comment:
As @costasovo mentioned, we already [have a fix](https://github.com/mailpoet/mailpoet/blob/62cf852691ec65370e559d5360339d92cf8b4e54/assets/css/src/components-form-editor/_block-editor.scss#L124-L125) for block tools to not hide under the top bar. While this PR fixes the problem, I see two issues here:

1. We are fixing the same problem in two places. This is not ideal, because it's harder to maintain. If we keep the margin, we should remove the padding fix.
2. This fix introduces another issue and that's unnecessary space above the sidebar. This can be solved by using existing selector `.edit-post-layout .interface-interface-skeleton__content` instead of `.interface-interface-skeleton__body`.
<img width="438" alt="Screenshot 2021-11-03 at 10 56 38" src="https://user-images.githubusercontent.com/470616/140040642-5903b0b6-0dc1-4079-a053-f6a3667b013e.png">

I'm fine with either of the solutions. Be it using margin or updating an already existing fix with bigger padding, but I wouldn't recommend using both.

Reply:
@NeosinneR Thanks for the feedback. Based on @costasovo's suggestion, I will remove the old walk around padding and unify the fix under the  `.edit-post-layout .interface-interface-skeleton__content` class.